support/mock/mockrt.go

```diff
@@ -209,15 +210,20 @@ func (rt *Runtime) Send(toAddr addr.Address, methodNum abi.MethodNum, params run
 	if len(rt.expectSends) == 0 {
 		rt.t.Fatalf("unexpected expectedMessage to: %v method: %v, value: %v, params: %v", toAddr, methodNum, value, params)
 	}
-
 	expectedMsg := rt.expectSends[0]
 
 	if expectedMsg.Equal(toAddr, methodNum, params, value) {
 		rt.t.Errorf("expectedMessage being sent does not match expectation. Message: to %v method: %v value: %v params: %v, Expected: %v", toAddr, methodNum, value, params, rt.expectSends[0].String())
 	}
-	// pop the expectedMessage from the queue
+
+	if value.GreaterThan(rt.balance) {
+		rt.Abort(exitcode.ErrInsufficientFunds, "cannot send value: %v exceeds balance: %v", value, rt.balance)
```

Comment:
Nit, but this should use `SysErrInsufficientFunds` (errors from within the runtime are "system errors")